README.md

```diff
@@ -0,0 +1,32 @@
+OregonDigital\_2
+===============
+
+[![Circle CI](https://circleci.com/gh/OregonDigital/oregondigital_2.svg?style=svg)](https://circleci.com/gh/OregonDigital/oregondigital\_2)
+
+
+Local Development Setup
+-----
+
+Requirements:
+
+  **Ruby Version: 2.0**
+
+Configuration and database creation:
+
+    git clone git@github.com:OregonDigital/oregondigital_2.git
+	cd oregondigital_2
+	bundle install
+	rake db:migrate
```

Comment:
It may just be my cross to bear, but is it worth adding "bundle exec " before these rake commands?  I get really odd errors when I use "rake" without aliasing it in bash or using binstub magic.